tests/test_anc.py

```diff
@@ -15,6 +15,11 @@
 
 from .utilities import get_interactive_context_state
 
+pytest.skip(
+    allow_module_level=True,
+    reason="Model 14 ANC and model 15 facility choice updates have obsoleted these tests.",
+)
```

Comment:
If these tests are obsolete (and will remain so), maybe we should remove them rather than skipping them?

Reply:
Sounds good to me - I wanted to make sure that we didn't want to update them instead of deleting.